bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -56,6 +56,22 @@
 			</add-channel>
 		</instruction-set>
 
+		<instruction-set targetVersion="1">
```

Comment:
Of course, I know that. But previous PR that already increased it was merged yesterday and I don't want to increase it again, especially when both PRs are very linked.
Note that the snapshot build failed this night. Do if this PR is merged today, we are sure noone will be impacted.

Reply:
In that case, can you merge it with the previous section, so that there is only one section like this:
```xml
<instruction-set targetVersion="1">
    ....
</instruction-set>
```

?

Reply:
Upgrade instructions for xdsl channels now grouped with those for ftth

Reply:
Just one last thing: Do you have any preferred order? Because I believe they will be added in exactly that order. So maybe you'd like to rearrange something to align with the order when creating a new Thing, now that we are merging the instructions anyway?

Reply:
Yes, the order is fine.